src/components/GuardianIdeal/GuardianIdealCarousel.tsx

```diff
@@ -55,6 +57,11 @@ const GuardianIdealCarousel = ({
             aria-label="previous guardian ideal"
             data-testid="slick-prev"
             onClick={() => {
+              trackEvent(
```

Comment:
Question: Can you add a test that ensures `trackEvent` has been called when we click? Same for below? I believe other tests do this.